File: 2729. Check if The Number is Fascinating/solution.java
class Solution {
  public boolean isFascinating(int n) {
    int timesTwo = n * 2;
    int timesThree = n * 3;
    String concat = String.valueOf(n) + String.valueOf(timesTwo) + String.valueOf(timesThree);
    Set<Character> concatSet = new HashSet<>();
    for (int index = 0; index < concat.length(); index++){
      concatSet.add(concat.charAt(index));
    }
    return (concatSet.size() == 9) && (concat.length() == 9) && (!concatSet.contains('0'));
  }
}

File: 2729. Check if The Number is Fascinating/solution.py
class Solution(object):
  def isFascinating(self, n):
    """
    :type n: int
    :rtype: bool
    """
    timesTwo = n * 2
    timesThree = n * 3
    concat = str(n) + str(timesTwo) + str(timesThree)
    concatSet = set(concat)
    return (len(concatSet) == 9) and (len(concat) == 9) and ('0' not in concatSet)